{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2018-11102",
    "ASSIGNER": "cve@mitre.org"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "CWE-119"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://docs.google.com/document/d/18xCwfxMSJiQ9ruQSVaO8-jlcobDjFiYXWOaw31V37xo/edit",
        "name": "https://docs.google.com/document/d/18xCwfxMSJiQ9ruQSVaO8-jlcobDjFiYXWOaw31V37xo/edit",
        "refsource": "MISC",
        "tags": [
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://bugzilla.libav.org/show_bug.cgi?id=1128",
        "name": "https://bugzilla.libav.org/show_bug.cgi?id=1128",
        "refsource": "MISC",
        "tags": [
          "Issue Tracking",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://lists.debian.org/debian-lts-announce/2019/09/msg00000.html",
        "name": "[debian-lts-announce] 20190902 [SECURITY] [DLA 1907-1] libav security update",
        "refsource": "MLIST",
        "tags": []
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "An issue was discovered in Libav 12.3. A read access violation in the mov_probe function in libavformat/mov.c allows remote attackers to cause a denial of service (application crash), as demonstrated by avconv."
      }
    ]
  }
}